src/Storages/System/StorageSystemTimeZones.generated.cpp

```diff
@@ -0,0 +1,565 @@
+// autogenerated by ClickHouse/contrib/cctz-cmake/CMakeLists.txt
```

Comment:
This should not be committed.

Reply:
I see -  interesting, I realized that the tests fail with `system.time_zones` table not found without committing this file. Besides files like the [StorageSystemContributors.generated.cpp](https://github.com/ClickHouse/ClickHouse/blob/master/src/Storages/System/StorageSystemContributors.generated.cpp), [StorageSystemBuildOptions.generated.cpp.in](https://github.com/ClickHouse/ClickHouse/blob/master/src/Storages/System/StorageSystemBuildOptions.generated.cpp.in) are being committed though. 

Though I realized that downloading the exact binary being build and running locally actually shows the systems.time_zones table all fine.